bindata/manifests/plugins/002-rbac.yaml

```diff
@@ -10,3 +10,61 @@ kind: ServiceAccount
 metadata:
   name: sriov-device-plugin
   namespace: {{.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Could the role be the same for both the cni and the dp and then bind both accounts to the same role?

Reply:
Technically, yes. 
@zshi-redhat do you think DP and CNI pod will need different roles in the future?

Reply:
@pliurh for the foreseeable future, I didn't see there is such need.